src/register/data/tests/reducers.test.js

```diff
@@ -102,6 +103,19 @@ describe('Registration Reducer Tests', () => {
     expect(reducer(state, action)).toEqual(state);
   });
 
+  it('should reset email error field data on focus of email field', () => {
+    const state = {
+      ...defaultState,
+      registrationError: { email: 'This email is already associated with an existing or previous Your Platform Name Here account' },
+    };
+    const action = {
+      type: CLEAR_REGISTER_BACKEND_ERROR,
+      payload: 'email',
+    };
+
+    expect(reducer(state, action)).toEqual(state);
```

Comment:
The error state should be null in the reducer while dispatching this action.

First, we need to set error state and then by dispatching our action, reducer state should be set to default.